src/components/Elements/ButtonMailto.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const ButtonMailto = ({ email, children }) => (
+  // eslint-disable-next-line jsx-a11y/anchor-is-valid
+  <Link
+    to="#"
+    onClick={(e) => {
+      window.open(`mailto:${email}`, '_blank');
+      e.preventDefault();
+    }}
+  >
+    {children}
+  </Link>
+);
+
+export default ButtonMailto;
```

Comment:
This suggestion fix the "jsx-a11y/anchor-is-valid" eslint error

```suggestion
import { Box } from '@mui/system';
import React from 'react';

const ButtonMailto = ({ email, children }) => (
  <Box
    sx={{ cursor: 'pointer' }}
    onClick={(e) => {
      window.open(`mailto:${email}`, '_blank');
      e.preventDefault();
    }}
  >
    {children}
  </Box>
);

export default ButtonMailto;
```

Reply:
Done, thanks! ✅